### PR TITLE
(BKR-1038) Expand fixture_contents to not care about file extensions

### PR DESCRIPTION
**Pull request by mchllweeks**

### CODE REVIEW

File: acceptance/lib/helpers/test_helper.rb [hunk 1]
```diff
@@ -43,9 +43,10 @@ def fixture_path
 end
 
 # Returns the contents of a named fixture file, to be found in `fixture_path`.
+# This will fail if there are multiple files with over lapping names.  ie sles-12-x86, sles-12-x86_64.
 def fixture_contents(fixture)
-  fixture_file = File.join(fixture_path, "#{fixture}.txt")
-  File.read(fixture_file)
+  fixture_file = Dir.entries(fixture_path).find { |e| /#{fixture}/ =~ e }
```
Comment (rick) [inline acceptance/lib/helpers/test_helper.rb]: We might want to anchor the regex here, or we risk getting the wrong fixture if one fixture's name is a substring of another's.  Maybe something like:

``` ruby
fixture_file = Dir.entries(fixture_path).find { |e| /^#{fixture}\./ =~ e }
```

Comment (mchllweeks) [inline acceptance/lib/helpers/test_helper.rb]: Thx for the improved regex.  I made the change to fixture_contents.

**Comment by mchllweeks** [comment]
retest this please

**Comment by tvpartytonight** [comment]
LGTM!

**Pull request merged**